workinggroups/serverless.md

```diff
@@ -0,0 +1,39 @@
+# CNCF Serverless WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore the intersection of cloud native and serverless technology.
```

Comment:
As above, "explore" is an activity, not an objective. Could we please craft an actual objective?

Reply:
Or eliminate the object subsections, if redundant with goals.